torch/_dynamo/eval_frame.py

```diff
@@ -642,6 +644,221 @@ def __bool__(self):
         )
 
 
+ConstraintSpec = namedtuple("ConstraintSpec", ["constraint_dim", "min_val", "max_val"])
```

Comment:
No constrain spec notions in eval frame.

Reply:
Hmm the export API takes in Constraint notion as input tho